build-info-client/src/main/java/org/jfrog/build/client/ArtifactoryHttpClient.java

```diff
@@ -60,19 +61,25 @@
     private final String artifactoryUrl;
     private final String username;
     private final String password;
+    private final String accessToken;
     private int connectionTimeout = DEFAULT_CONNECTION_TIMEOUT_SECS;
     private int connectionRetries = DEFAULT_CONNECTION_RETRY;
     private ProxyConfiguration proxyConfiguration;
 
     private PreemptiveHttpClient deployClient;
 
-    public ArtifactoryHttpClient(String artifactoryUrl, String username, String password, Log log) {
+    public ArtifactoryHttpClient(String artifactoryUrl, String username, String password, String accessToken, Log log) {
```

Comment:
Any reason why we need a constructor that gets both username/password and an access token? This is reflected also in other constructors in the code.